.github/workflows/docker_publish.yml

```diff
@@ -54,70 +54,16 @@ jobs:
         uses: firehed/multistage-docker-build-action@v1
         with:
           repository: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-${{ matrix.compiler}}-ext-hdf5_${{ matrix.hdf5_versions}}-moab_${{ matrix.moab_versions }}
-          stages: base, external_deps, hdf5
-          server-stage: moab
-          quiet: false
-          parallel: true
-          tag-latest-on-default: ${{ env.tag-latest-on-default }}
-          dockerfile: CI/Dockerfile
-          build-args: COMPILER=${{ matrix.compiler }}, UBUNTU_VERSION=${{ matrix.ubuntu_versions }}, HDF5=${{ matrix.hdf5_versions }}, MOAB=${{ matrix.moab_versions }}
-
-  build-dagmc_test-img:
-    needs: [build-dependency-img]
-    runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        ubuntu_versions : [
-          20.04,
-          22.04,
-          ]
-        compiler : [
-          gcc,
-          clang,
-          ]
-        hdf5_versions : [
-         1.10.4,
-        ]
-        moab_versions : [
-         5.3.0,
-        ]
-
-    name: Installing DAGMC
-    steps:        
-      - name: default environment
-        run: |
-          echo "tag-latest-on-default=false" >> "$GITHUB_ENV"
-
-      - name: condition on trigger parameters
-        if: ${{ github.repository_owner == 'svalinn' && github.ref == 'refs/heads/develop' }}
-        run: |
-          echo "tag-latest-on-default=true" >> "$GITHUB_ENV"
-
-      - name: Log in to the Container registry
-        uses: docker/login-action@v2
-        with:
-          registry: ghcr.io
-          username: ${{ github.repository_owner }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Checkout repository
-        uses: actions/checkout@v3
-
-      - name: Build & test DAGMC in Docker image
-        uses: firehed/multistage-docker-build-action@v1
-        with:
-          repository: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-${{ matrix.compiler }}-ext-hdf5_${{ matrix.hdf5_versions }}-moab_${{ matrix.moab_versions }}
-          stages: moab, dagmc
+          stages: base, external_deps, hdf5, moab, dagmc
           server-stage: dagmc_test
           quiet: false
           parallel: true
           tag-latest-on-default: ${{ env.tag-latest-on-default }}
           dockerfile: CI/Dockerfile
-          build-args: COMPILER=${{ matrix.compiler }}, UBUNTU_VERSION=${{ matrix.ubuntu_versions }}, HDF5=${{ matrix.hdf5_versions }}, MOAB=${{ matrix.moab_versions }}
+          build-args: COMPILER=${{ matrix.compiler }}, UBUNTU_VERSION=${{ matrix.ubuntu_versions }}, HDF5_VERSION=${{ matrix.hdf5_versions }}, MOAB_BRANCH=${{ matrix.moab_versions }}
```

Comment:
One quick thought after looking more closely at the Dockerfile -- do we need to set the `HDF5_VERSION_major` arg here as well? There's a note in the Dockerfile that it needs to be set manually.

Reply:
Wow! Great catch - I'll have to dig into that.  From a quick skim, I think we can automate that - will look tonight/tomorrow

Reply:
Was able to make this work in latests commit